### DISCUSSION TITLE
app_store_connect_api_key - invalid curve name - GitHub Actions

### DISCUSSION THREAD

**Discussion by ghost** [fastlane/fastlane#18017] [Q&A] [answered]
i use the following
```
  check_app_store_metadata(
    api_key: app_store_connect_api_key(
      duration: 1200,
      in_house: false,
      is_key_content_base64: true
    )
  )
```
with the Enviroments:
`APP_STORE_CONNECT_API_KEY_ISSUER_ID`
`APP_STORE_CONNECT_API_KEY_KEY`
`APP_STORE_CONNECT_API_KEY_KEY_ID`

it works local! 👍 

but if i try to use this via github actions i get
`invalid curve name\e[0m (OpenSSL::PKey::ECError)`

how can i use `app_store_connect_api_key` with github actions?

**Comment by dinsen** [comment]
Hi

Maybe this can help you?

https://github.com/fastlane/fastlane/issues/17340

**Comment by ghost** [comment, marked as answer]
my bad. i used a envorment on github .. i need to use a secret below the environment section
